clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -4526,7 +4526,7 @@ def err_objc_attr_typedef_not_void_pointer : Error<
 def err_objc_cf_bridged_not_interface : Error<
   "CF object of type %0 is bridged to %1, which is not an Objective-C class">;
 def err_objc_ns_bridged_invalid_cfobject : Error<
-  "ObjectiveC object of type %0 is bridged to %1, which is not valid CF object">;
+  "Objective-C object of type %0 is bridged to %1, which is not valid CF object">;
```

Comment:
I've found `ObjectiveC` still exists in elsewhere:
https://github.com/llvm/llvm-project/blob/5a81db311cf8e15b7c6a33100bb5c1fb256754ce/clang/include/clang/Basic/DiagnosticParseKinds.td#L1120
https://github.com/llvm/llvm-project/blob/5a81db311cf8e15b7c6a33100bb5c1fb256754ce/clang/include/clang/Basic/DiagnosticSemaKinds.td#L4529

Reply:
Good catch, I've fixed it up